sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+## Material Change
+
+Wendy Chun: "Richard Stallman, in his critique of nonfree software, has argued that an executable program 'is a mysterious bunch of numbers. What it does is secret.' Against this magical execution, source code supposedly provides an understanding and a freedom--the ability to map and know the workings of the machine, but, again, only through a magical erasure of the gap between source and execution, an erasure of execution itself" (51).
+
+This Cartesian dualism that Chun refers to, source/execution, haunts computation to its core. It is implicated in a related binary that Chun overlooks, the separation of _code_ from _data_. These dualisms crucially influence the way in which software's form and potential is commonly understood today, and account for how its materiality is often overlooked or fails to actualize at all. Conventional source code, before it can be used, must be transformed into something different: a binary executable created by a _compiler_. This transformation is unidirectional; as Chun suggests, it is an act of erasure that enacts the practical gap between source code, which is understood as fundamentally changeable and authorable--the "source" of creative power--and execution, which freezes the creative act and transforms it into an object and product. While code is apparently iterative and endlessly modifiable (by those with the skills and access, anyway), executables are not. One cannot unravel a piece of software and turn it into something else.
+
+Software has no equivalent to Amy Twigger Holroyd's _reknitting_, where existing knitted garments (most strikingly, even those that were mass-produced by machines) are modified or transformed in ways that were unanticipated by their original designers. Re-knitting, according to Twigger Holroyd, involves "a broad range of processes which can be carried out by individuals to alter existing items of knitwear. Re-knitting shifts a knitter's practice from the creation of new items to the re-making of existing pieces" ("About Re-Knitting"). These processes include unravelling, cutting, grafting, insertions, stitch hacks, and replacements (Twigger Holroyd, "The Spectrum"). With these techniques, Twigger Holroyd and her friends make marvelously uncanny and expressive machine/hand hybrid garments by grafting new designs into old jumpers and socks.
```

Comment:
This topic is clearly in your wheel house, but I think the conceptual work is now too and it shows. (I would say that this means there are a few places where you could slow down a bit… the initial transition to re-knitting, for example, is a little quick).